### ISSUE TITLE
NoMethodError: undefined method `descends_from_active_record?' for :foo:Symbol

### ISSUE THREAD

**Issue by lorenzk** [CanCanCommunity/cancancan#676]
### Steps to reproduce

Here is an [example repo](https://github.com/lorenzk/cancancan-symbol-bug) but the important parts are:

```ruby
# app/models/ability.rb
can :manage, [:foo, Post] # has to be a mixed array with symbols and classes
```

```ruby
# app/controllers/posts_controller.rb
class PostsController < ApplicationController
  load_and_authorize_resource
  # ...
end
```

### Expected behavior
cancancan should realize `:foo` is a symbol and not an active record class.

### Actual behavior
```
bin/rails test --backtrace
Running via Spring preloader in process 56538
Run options: --backtrace --seed 36622

# Running:

...E

Error:
PostsControllerTest#test_should_get_index:
NoMethodError: undefined method `descends_from_active_record?' for :foo:Symbol
    /Users/lorenz/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/cancancan-3.2.0/lib/cancan/model_adapters/sti_normalizer.rb:11:in `block (2 levels) in normal
ize'
    /Users/lorenz/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/cancancan-3.2.0/lib/cancan/model_adapters/sti_normalizer.rb:10:in `select'
    /Users/lorenz/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/cancancan-3.2.0/lib/cancan/model_adapters/sti_normalizer.rb:10:in `block in normalize'
    /Users/lorenz/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/cancancan-3.2.0/lib/cancan/model_adapters/sti_normalizer.rb:9:in `delete_if'
    /Users/lorenz/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/cancancan-3.2.0/lib/cancan/model_adapters/sti_normalizer.rb:9:in `with_index'
    /Users/lorenz/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/cancancan-3.2.0/lib/cancan/model_adapters/sti_normalizer.rb:9:in `normalize'
    /Users/lorenz/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/cancancan-3.2.0/lib/cancan/model_adapters/active_record_adapter.rb:17:in `initialize'
    /Users/lorenz/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/cancancan-3.2.0/lib/cancan/ability.rb:172:in `new'
    /Users/lorenz/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/cancancan-3.2.0/lib/cancan/ability.rb:172:in `model_adapter'
    /Users/lorenz/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/cancancan-3.2.0/lib/cancan/model_additions.rb:24:in `accessible_by'
    /Users/lorenz/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/cancancan-3.2.0/lib/cancan/controller_resource_loader.rb:103:in `load_collection'
    /Users/lorenz/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/cancancan-3.2.0/lib/cancan/controller_resource_loader.rb:20:in `load_resource'
    /Users/lorenz/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/cancancan-3.2.0/lib/cancan/controller_resource.rb:33:in `load_and_authorize_resource'
    /Users/lorenz/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/cancancan-3.2.0/lib/cancan/controller_resource.rb:17:in `block in add_before_action'
```

### System configuration
**Rails version**: Rails 6.0.3.4

**Ruby version**: ruby 2.7.1p83

**CanCanCan version**: cancancan (3.2.0)

**Comment by coorasse** [comment]
closed in 3.2.1

**Comment by lorenzk** [comment]
Thank you! 👍
